src/Options/Definitions.js

```diff
@@ -876,3 +876,11 @@ module.exports.DatabaseOptions = {
     default: false,
   },
 };
+module.exports.AuthAdapter = {
+  enabled: {
+    env: 'undefinedENABLED',
```

Comment:
So this also generates an env var, which is of course unnecessary because our auth adapters don't take any. The larger issue is that adapters completely lack definitions. We may have to introduce a new var here to suppress generating an env var for this param. Can you try to set an empty string as env var?

Reply:
Did that actually work? Setting `:ENV:` does not generate an entry for `env:`?

Reply:
Yep!

Reply:
Cool